scripts/build-and-package-windows.ps1

```
@@ -47,6 +47,62 @@ function Write-Sha256Sidecar($ArtifactPath) {
+    return @()
+  }
+  return ($Value -split '[,;\s]+' | Where-Object { -not [string]::IsNullOrWhiteSpace($_) })
+}
+
+function Invoke-F4LinterEvidenceHook($ArtifactId) {
+  $mode = if ([string]::IsNullOrWhiteSpace($env:ECLI_F4_LINTER_PROVISIONING_MODE)) {
+    "dry-run"
+  } else {
+    $env:ECLI_F4_LINTER_PROVISIONING_MODE
+  }
+  $profile = if ([string]::IsNullOrWhiteSpace($env:ECLI_F4_LINTER_PROFILE)) {
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'profile' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)